client/reader/following/main.jsx

```diff
@@ -43,7 +43,7 @@ function handleSearch( query ) {
 	}
 }
 
-const lastDayForVoteBanner = new Date( '2018-11-07T00:00:00' );
+const lastDayForVoteBanner = new Date( '2020-10-18T00:00:00' );
```

Comment:
i think its the 17th, no?

Reply:
I _think_ this date was choosen so the message hides at the very end of the 17th.

[Polls are open till 7pm](https://vote.nz/voting/ways-you-can-vote/find-a-voting-place-on-a-map/) so using `2020-10-17T19:00:00` is another option.

Reply:
It is! The variable is poorly named - I'll change it. It's the time at which the banner is hidden.

For example, here it is for today:

<img width="649" alt="Screen Shot 2020-10-13 at 14 14 07" src="https://user-images.githubusercontent.com/17325/95804247-9b635f80-0d5e-11eb-9a9e-53f81ded51f0.png">

Reply:
Renamed to `hideVoteBannerDate`.

Reply:
Also changed to 7pm on 17th as per @p-jackson's suggestion 👍